README.md

```diff
@@ -34,74 +34,39 @@ sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
 kubectl apply -f ./kubemacpool.yaml
 ```
 
-### Opting-in to kubemacpool service
+### kubemacpool service
 
-Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside only on opted-in namespaces. You can opt-in your namespace by adding the following labels:
-- `mutatepods.kubemacpool.io=allocate` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt in vms mac allocation in your namespace
+Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside on all namespaces (opt-out). You can opt-out your namespace by adding the following labels:
```

Comment:
the `(opt-out)` is not needed is just `on all namespaces.`

Also I would say that you can add here just a link to "How to opt-out" feels redundant to have the labels here and also on the "How to"

Reply:
Done